### PR TITLE
Add summary to README.md.

### PR DESCRIPTION
**Pull request by StevenYCChou**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Stackdriver Kubernetes Configs
 
+A collection of kubernetes configurations to integrate with Stackdriver
+products. For now, it supports [Stackdriver Logging](https://cloud.google.com/logging/)
+and [Stackdriver Monitoring](https://cloud.google.com/monitoring/).
+
```
Comment (bmoyles0117) [inline README.md]: Can we add a link to the documentation as well? 

https://cloud.google.com/monitoring/kubernetes-engine/customizing

Comment (StevenYCChou) [reply README.md]: How should I describe it? I noticed there are 2 docs [Installing Kubernetes Monitoring](https://cloud.google.com/monitoring/kubernetes-engine/installing) and [Manual Installation of Kubernetes Monitoring](https://cloud.google.com/monitoring/kubernetes-engine/customizing). I can see the [Manual Installation of Kubernetes Monitoring](https://cloud.google.com/monitoring/kubernetes-engine/customizing) use the the content of this repo. On the other hand, [Installing Kubernetes Monitoring](https://cloud.google.com/monitoring/kubernetes-engine/installing) just use gcloud command.

Does [Installing Kubernetes Monitoring](https://cloud.google.com/monitoring/kubernetes-engine/installing) utilize this repo as well? If yes, I may expand the paragraph too.

Comment (qingling128) [reply README.md]: This repo is used by https://cloud.google.com/monitoring/kubernetes-engine/customizing only.

The other doc uses the Kubernetes OSS addon: https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/fluentd-gcp

File: README.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Stackdriver Kubernetes Configs
 
+A collection of kubernetes configurations to integrate with Stackdriver
+products. For now, it supports [Stackdriver Logging](https://cloud.google.com/logging/)
+and [Stackdriver Monitoring](https://cloud.google.com/monitoring/). This repo is only used
+for [manual installation](https://cloud.google.com/monitoring/kubernetes-engine/customizing) 
+on existing cluster.
```
Comment (igorpeshansky) [inline README.md]: Nit: "clusters".

Comment (StevenYCChou) [reply README.md]: updated

File: README.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
 # Stackdriver Kubernetes Configs
 
+A collection of kubernetes configurations to integrate with Stackdriver
```
Comment (igorpeshansky) [inline README.md]: Another nit: "Kubernetes".

**Pull request merged**